### DISCUSSION TITLE
Disable Social Share Buttons

### DISCUSSION THREAD

**Discussion by osterix-droid** [consuldemocracy/consuldemocracy#5653] [Q&A] [answered]
Hi, for an internal project, I would like to not show the social share buttons in discussions, proposals and votings. 
Can anybody advise, what is the best /recommend way to do this?
Many thanks in advance!

**Comment by javierm** [comment, marked as answer]
Hi, @osterix-droid :smile:.

To do that, you'll have to add some custom code. In this case, probably the easiest way would be creating an `app/views/custom/shared/` folder and creating an **empty** `app/views/custom/shared/_social_share.html.erb` file :thinking:. This will overwrite the contents of the original file that renders the social share buttons and, since it's empty, it will render nothing.

See also the [customization documentation](https://docs.consuldemocracy.org/tech_docs/customization).

Hope that helps!

**Comment by osterix-droid** [comment]
Hi @javierm cool, generic solution, no code required... and works! Muchas gracias ;)
